src/Module.Server/Common/CrpgPeer.cs

```diff
@@ -66,7 +74,7 @@ private void SynchronizeUserToEveryone()
         }
 
         GameNetwork.BeginBroadcastModuleEvent();
-        GameNetwork.WriteMessage(new UpdateCrpgUser { Peer = Peer, User = _user });
+        GameNetwork.WriteMessage(new UpdateCrpgUser { Peer = Peer, User = _user, ClanName = Clan?.Name ?? string.Empty });

```

Comment:
ClanTag is missing - currently results in an empty string if the peer has a clan name > 10 characters.